mapie/tests/test_time_series_regression.py

```diff
@@ -0,0 +1,357 @@
+    y_pred_array, y_pis_array = mapie_ts_reg.predict(X, alpha=alpha)
+    np.testing.assert_allclose(y_pred_float1, y_pred_array)
+    np.testing.assert_allclose(y_pred_float2, y_pred_array)
+    np.testing.assert_allclose(y_pis_float1[:, :, 0], y_pis_array[:, :, 0])
+    np.testing.assert_allclose(y_pis_float2[:, :, 0], y_pis_array[:, :, 1])
+
+
+@pytest.mark.parametrize("strategy", [*STRATEGIES])
+def test_results_for_ordered_alpha(strategy: str) -> None:
+    """
+    Test that prediction intervals lower (upper) bounds give
+    consistent results for ordered alphas.
+    """
+    mapie = MapieTimeSeriesRegressor(**STRATEGIES[strategy])
+    mapie.fit(X, y)
+    y_pred, y_pis = mapie.predict(X, alpha=[0.05, 0.1])
+    assert np.all(
+        np.abs(y_pis[:, 1, 0] - y_pis[:, 0, 0])
+        >= np.abs(y_pis[:, 1, 1] - y_pis[:, 0, 1])
+    )
+
+
+@pytest.mark.parametrize("strategy", [*STRATEGIES])
+def test_results_single_and_multi_jobs(strategy: str) -> None:
+    """
+    Test that MapieTimeSeriesRegressor gives equal predictions
+    regardless of number of parallel jobs.
+    """
+    mapie_single = MapieTimeSeriesRegressor(n_jobs=1, **STRATEGIES[strategy])
+    mapie_multi = MapieTimeSeriesRegressor(n_jobs=-1, **STRATEGIES[strategy])
+    mapie_single.fit(X_toy, y_toy)
+    mapie_multi.fit(X_toy, y_toy)
+    y_pred_single, y_pis_single = mapie_single.predict(X_toy, alpha=0.2)
+    y_pred_multi, y_pis_multi = mapie_multi.predict(X_toy, alpha=0.2)
+    np.testing.assert_allclose(y_pred_single, y_pred_multi)
+    np.testing.assert_allclose(y_pis_single, y_pis_multi)
+
+
+@pytest.mark.parametrize("strategy", [*STRATEGIES])
+def test_results_with_constant_sample_weights(strategy: str) -> None:
+    """
+    Test predictions when sample weights are None
+    or constant with different values.
+    """
+    n_samples = len(X)
+    mapie0 = MapieTimeSeriesRegressor(**STRATEGIES[strategy])
+    mapie1 = MapieTimeSeriesRegressor(**STRATEGIES[strategy])
+    mapie2 = MapieTimeSeriesRegressor(**STRATEGIES[strategy])
+    mapie0.fit(X, y, sample_weight=None)
+    mapie1.fit(X, y, sample_weight=np.ones(shape=n_samples))
+    mapie2.fit(X, y, sample_weight=np.ones(shape=n_samples) * 5)
+    y_pred0, y_pis0 = mapie0.predict(X, alpha=0.05)
+    y_pred1, y_pis1 = mapie1.predict(X, alpha=0.05)
+    y_pred2, y_pis2 = mapie2.predict(X, alpha=0.05)
+    np.testing.assert_allclose(y_pred0, y_pred1)
+    np.testing.assert_allclose(y_pred1, y_pred2)
+    np.testing.assert_allclose(y_pis0, y_pis1)
+    np.testing.assert_allclose(y_pis1, y_pis2)
+
+
+@pytest.mark.parametrize("method", ["enbpi"])
+@pytest.mark.parametrize("cv", [-1, 2, 3, 5])
+@pytest.mark.parametrize("agg_function", ["mean", "median"])
+@pytest.mark.parametrize("alpha", [0.05, 0.1, 0.2])
+def test_prediction_agg_function(
+    method: str, cv: Union[LeaveOneOut, KFold], agg_function: str, alpha: int
+) -> None:
+    """
+    Test that predictions differ when ensemble is True/False,
```

Comment:
The function does not test that `y_pis_1` and `y_pis_2` are similar ?

Reply:
PIs are the same but predictions differ

Reply:
I corrected